orttraining/orttraining/training_ops/cpu/nn/layer_norm.cc

```diff
@@ -11,34 +11,25 @@ namespace contrib {
 
 // LayerNormGrad
 
-#define REGISTER_KERNEL_TYPED(T)                                  \
-  ONNX_OPERATOR_TYPED_KERNEL_EX(                                  \
-      LayerNormalizationGrad,                                     \
-      kMSDomain,                                                  \
-      1,                                                          \
-      T,                                                          \
-      kCpuExecutionProvider,                                      \
-      KernelDefBuilder()                                          \
-          .TypeConstraint("T", DataTypeImpl::GetTensorType<T>()), \
-      LayerNormGrad<T, false>);                                   \
-  ONNX_OPERATOR_TYPED_KERNEL_EX(                                  \
-      SimplifiedLayerNormalizationGrad,                           \
-      kMSDomain,                                                  \
-      1,                                                          \
-      T,                                                          \
-      kCpuExecutionProvider,                                      \
-      KernelDefBuilder()                                          \
-          .TypeConstraint("T", DataTypeImpl::GetTensorType<T>()), \
-      LayerNormGrad<T, true>);                                    \
-  ONNX_OPERATOR_TYPED_KERNEL_EX(                                  \
-      InvertibleLayerNormalizationGrad,                           \
-      kMSDomain,                                                  \
-      1,                                                          \
-      T,                                                          \
-      kCpuExecutionProvider,                                      \
-      KernelDefBuilder()                                          \
-          .TypeConstraint("T", DataTypeImpl::GetTensorType<T>()), \
-      InvertibleLayerNormGrad<T>);
+#define REGISTER_KERNEL_TYPED(T)                                                                          \
```

Comment:
Is this the new clang-formatting?

Reply:
Using the .clang-format file and the VSCode's "Format document/selection", the code will be formatted like this. It will try to use the 120 columns as possible as it can instead of starting a new line for each argument or function caller. So the answer is yes, or maybe the old code just not the correct clang-format in the .clang-format file.